<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dept</class>
 <widget class="QWidget" name="dept">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>925</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>891</width>
     <height>731</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableWidget" name="tw_dept">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>871</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="dept_group_box">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>491</width>
       <height>380</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>491</width>
       <height>380</height>
      </size>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>부서명</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="deptName_le"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>담당자명</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="manName_le"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>권한</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="per_combo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>아이디</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dept_user_id"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>패스워드</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dept_user_pass_le"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="reg_dept_btn">
          <property name="text">
           <string>등록</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="update_dept_btn">
          <property name="text">
           <string>수정</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_dept_btn">
          <property name="text">
           <string>삭제</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="res_le"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
